packages/zoe/src/contracts/loan/borrow.js

```diff
@@ -15,14 +15,22 @@ import { makeAddCollateralInvitation } from './addCollateral';
 /** @type {MakeBorrowInvitation} */
 export const makeBorrowInvitation = (zcf, config) => {
   const {
-    mmr, // Maintenance Margin Requirement, in percent
+    mmr, // Maintenance Margin Requirement, as a Percent (deprecated)
     priceAuthority,
     periodNotifier,
     interestRate,
     interestPeriod,
     lenderSeat,
   } = config;
 
+  // TODO(hibbert) drop mmr as Percent before Beta
+  let {
+    mmrRatio, // Maintenance Margin Requirement, as a ratio
+  } = config;
+  if (!mmrRatio) {
+    mmrRatio = mmr.makeRatio();
+  }
+
```

Comment:
borrow is internal to the loan contract, so it doesn't make sense to pass around mmr - we should have already changed it to a Ratio at this point, after hitting index. However, I think it's best to just drop the percent and use ratios directly anyways.